From c17484448663f9b2d4148faa596a077d0ae79891 Mon Sep 17 00:00:00 2001
From: Lili Li <lili.li@intel.com>
Date: Wed, 30 Jun 2021 12:05:42 +0800
Subject: [PATCH 1/2] drm: hantro: rename and cleanup dma_resv_get_excl v3

API changed since commit 8195e3c78b28 ("dma-buf: rename and cleanup dma_resv_get_excl v3
")

Signed-off-by: Lili Li <lili.li@intel.com>
Signed-off-by: Pan, Kris <kris.pan@intel.com>
---
 drivers/gpu/drm/hantro_driver/hantro_drm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/hantro_driver/hantro_drm.c b/drivers/gpu/drm/hantro_driver/hantro_drm.c
index b63c66613179..5f1c6fa1b071 100644
--- a/drivers/gpu/drm/hantro_driver/hantro_drm.c
+++ b/drivers/gpu/drm/hantro_driver/hantro_drm.c
@@ -940,7 +940,7 @@ static int hantro_test(struct drm_device *dev, void *data,
 	if (!obj)
 		return -EINVAL;
 
-	pfence = dma_resv_get_excl(obj->dma_buf->resv);
+	pfence = dma_resv_excl_fence(obj->dma_buf->resv);
 	while (ret > 0)
 		ret = schedule_timeout(ret);
 
-- 
2.27.0

